### PR TITLE
add keep parameter

### PR DESCRIPTION
**Pull request by zkamvar**
I am in a situation where I want to archive a certain subset of repositories before I make irreversible changes to them.

This keep parameter will help me avoid downloading hundreds of repositories that I do not wish to archive.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # gitcellar 0.0.0.9000
 
+* Parameter `keep` allows users to define which repositories should be archived
```
Comment (maelle) [inline NEWS.md]: Please add your username here :smile_cat:

**Review by maelle — CHANGES_REQUESTED** [review]
Thanks a ton! Two small comments.

File: NEWS.md (hunk 1 above)
Comment (zkamvar) [reply NEWS.md]:
```suggestion
* Parameter `keep` allows users to define which repositories should be archived (#13, @zkamvar)
```

**Comment by zkamvar** [comment]
Done! I think? I only saw one comment

**Comment by maelle** [comment]
Oops, my other comment was, what should happen is some of the "keep" names are NOT in the repos? Should there be a warning?

**Comment by zkamvar** [comment]
> Oops, my other comment was, what should happen is some of the "keep" names are NOT in the repos? Should there be a warning?

Oh! Good point! I'll add that logic in and throw an error because the keep parameter is opt-in, so the user will likely want to know immediately if they are requesting repositories that are not passed through for a process that could take a long time.

**Comment by zkamvar** [comment]
This is ready for another round of review when you are back :)

**Comment by maelle** [comment]
Thank you!!

**Pull request merged**